src/material-examples/typings.d.ts

```diff
@@ -0,0 +1 @@
+declare var module: {id: string};
```

Comment:
It was part of my debugging process. It ended up not being the problem, but I left it because the material2 components themselves use it. However, I didn't think of the plunkers. It looks like I failed to update how those work at all. Fixing that now...